napari/plugins/pypi.py

```diff
@@ -129,62 +117,6 @@ def ensure_published_at_pypi(
     )
 
 
-@lru_cache(maxsize=128)
-def ensure_repo_is_napari_plugin(
```

Comment:
Given that we're dropping the GitHub `repo` search it's fine to drop this method, but I wonder if we might want to keep some of the `parser.has_option('options.entry_points', 'napari.plugin')` and apply that to what we surface on PyPI using the framework classifier. That can come in future PRs, but wanted to mention it now too. cc @tlambert03

Reply:
we don't currently look at the source of anything we surface at PyPI, and it would be a bit harder to do so.  I think it's just safe to include anything that says framework :: napari.  Worst case scenario, they don't offer an entry_point and it's an "implementation-free plugin"

Reply:
(so, I'm fine just killing all of this.  it was a workaround to begin with)